modules/cython/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://cython.org/",
+    "maintainers": [],
```

Comment:
Do you mind adding yourself as maintainer for this module?

Reply:
I would prefer to keep it empty in order to avoid becoming the bottleneck to approve PRs. Furthermore, I don't really know or use this module, only uploading as transitive dependency.

Reply:
Done